### ISSUE TITLE
ccs.bam conversion and tools options

### ISSUE THREAD

**Issue by Overcraft90** [chhylp123/hifiasm#279]
Hi there,

I'm currently dealing with some _**ccs.bam**_ files and I was wondering first, if it is necessary to convert them to _**fasta**_ or _**fastq**_ for `hifiams` to handle those data; secondly, I was wondering if for the conversion I can use anything other than `bam2fastq`, which for some reasons I could not install... Let me know.

Thanks

**Comment by chhylp123** [comment]
Yes, hifiasm does not accept bam files. It seems `samtools fastq` can also conduct the conversion?

**Comment by Overcraft90** [comment]
Thanks,

That is great I will give it a try and let you know.

**Comment by Overcraft90** [comment]
Hi and thanks,

I have been able to successfully retain only reads with QV score ≥ 30 by using `samtools fastq`. I think we can close the issue.

**Comment by chhylp123** [comment]
Thanks.
